rudder-web/src/main/scala/com/normation/rudder/web/rest/node/NodeAPIService8.scala

```diff
@@ -138,54 +128,32 @@ class NodeApiService8 (
     }
   }
 
-  private[this] class RunHandler (out : Option[OutputStream]) extends NuAbstractProcessHandler {
-
-    // set output (stdout or stderr) buffer to our stream
-    // Synchronized so outputs are not mixed
-    def onOut(buf: ByteBuffer, isClosed : Boolean) = {
-      out match {
-        case Some(out) =>
-          val bytes : Array[Byte] = new Array(buf.remaining())
-          buf.get(bytes)
-          out.synchronized {
-            out.write(bytes)
-            out.flush()
-          }
-        case None => // Nothing to do for now
-      }
-    }
-
-    // On exit will be called at the end of the process, or if any error/exception occurs, so we can close our outputstream here
-    override def onExit( statusCode : Int) = {
-      out.map(_.close())
-    }
+  def remoteRunRequest(nodeId: NodeId, classes : List[String], keepOutput : Boolean, asynchronous : Boolean) = {
+    val url = s"https://localhost/rudder/relay-api/remote-run/nodes/${nodeId.value}"
```

Comment:
This should be configured. I'm sure there will be case where relay-api won't be on the same machine than the web app (or behind an other root)